loc/US/nomads_strings_core.lua

```diff
@@ -68,15 +68,15 @@ tooltip_lobby_unit_restriction_nonomads_desc = ""
 tooltip_radar_boost_title = "Radar Boost Toggle"
 tooltip_radar_boost_desc = "Starts/stops charging radar boosting for the selection units. When finished charging up radar boost initiates automatically. Charging up requires energy."
 tooltip_bombardmode_title = "Bombard Mode Toggle"
-tooltip_bombardmode_desc = "Turn the selection units bombard mode on/off"
+tooltip_bombardmode_desc = "bombard mode decrese reload time by 50% but increse firing randomnes"
 tooltip_no_air_title = "Anti-Air Toggle"
 tooltip_no_air_desc ="Turn the selection units anti-air attack on/off"
 tooltip_snipermode_title = "Sniper Mode Toggle"
 tooltip_snipermode_desc = "Turn sniper mode on/off"
 tooltip_usecapacitor_title = "Capacitor Toggle"
 tooltip_usecapacitor_desc = "Boost unit temporarily (firepower, build power, regeneration) using the capacitor ability"
 tooltip_anchor_title = "Anchor Toggle"
-tooltip_anchor_desc = "Immobilizes the unit"
+tooltip_anchor_desc = "Immobilizes the unit and increse weapon range, vision, radar and build rate by 30%"
```

Comment:
fix spelling:
"Immobilizes the unit and increases weapon range, vision, radar and build rate by 30%."